tests/remotes_env.sample

```diff
@@ -0,0 +1,13 @@
+# Uncomment and fill in to test against S3
```

Comment:
I don't think that this file is necessary at all. This should be described in docs, not in some random `*.sample` file that no one will be able to ever find.

Reply:
It would have saved me a couple of days, since now you need to carefully read test code, ci scripts and make guesses to make tests run.

Reply:
Even with all the env vars documented it is still a pain to track all of them, since there are so many. This file provides a template, which will save time and standardize how env vars set and help with their exchange within a team - starting from our own.

Reply:
So you do have this file referenced in the https://dvc.org/doc/user-guide/contributing#testing-remotes . I'm ok with it then.